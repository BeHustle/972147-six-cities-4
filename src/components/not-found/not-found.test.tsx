import * as React from 'react';
import * as renderer from 'react-test-renderer';
import NotFound from './not-found';
import {Router} from 'react-router-dom';
import {history} from '../../history';


it(`Render notFound screen`, () => {
  const tree = renderer
    .create(<Router history={history}>
      <NotFound />
    </Router>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
