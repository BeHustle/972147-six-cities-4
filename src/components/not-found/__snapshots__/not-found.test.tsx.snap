// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render notFound screen 1`] = `
<h1>
  Page not found. 
  <a
    href="/"
    onClick={[Function]}
  >
    Go to main
  </a>
</h1>
`;
